securesignals-ima-devapp/src/main/res/layout/activity_my.xml

```diff
@@ -0,0 +1,48 @@
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Nit: Naming wise, maybe change to match what activity it's for, e.g. `main_activity.xml`.

Reply:
Done.